modules/platforms/dotnet/Apache.Ignite/Internal/Proto/BinaryTuple/BinaryTupleBuilder.cs

```diff
@@ -534,6 +518,126 @@ public void AppendObject(object? value, ClientDataType colType, int scale)
             }
         }
 
+        /// <summary>
+        /// Appends an object.
+        /// </summary>
+        /// <param name="value">Value.</param>
+        public void AppendObjectWithType(object? value)
+        {
+            switch (value)
+            {
+                case null:
+                    AppendNull();
+                    AppendNull();
+                    AppendNull();
```

Comment:
The same comment as for Java. Also, there is some mismatch to Java packing for `null`

Reply:
Fixed.